apps/src/netsim/NetSimRouterLogModal.js

```diff
@@ -2,10 +2,12 @@
  * @overview a modal dialog showing the union of all router logs for the
  *           current shard.
  */
-'use strict';
```

Comment:
Now that we use Babel and treat everything as ES6, this is redundant because [ES6 modules are always in strict mode](http://stackoverflow.com/a/31685340/5000129).

Actually, I should do a cleanup PR that goes through and removes 'use strict' throughout our JavaScript.